data/json/recipes/recipe_deconstruction.json

```diff
@@ -69,13 +97,29 @@
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
     "components": [ [ [ "copper", 4 ] ] ]
   },
+  {
+    "result": "copper_pan",
+    "type": "uncraft",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "7 m 30 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "components": [ [ [ "scrap_copper", 4 ] ], [ [ "scrap", 1 ] ] ]
```

Comment:
This should yield more based upon weight. The pan weighs 628 g and the cumulative weight of its components is only 225 g.

Reply:
Similar to the iron pan, however, it theoretically only has 50ml of components while the components listed here would put it at 273ml.

Reply:
The id: "pan" entry seems wrong. It has 2628g, holds 1000ml but has only 1050ml volume, so I will increase it's volume and disassemble resulting components to match it's base weight.